
T:  Bus=01 Lev=01 Prnt=01 Port=01 Cnt=01 Dev#=  2 Spd=12  MxCh= 0
D:  Ver= 1.10 Cls=00(>ifc ) Sub=00 Prot=00 MxPS=64 #Cfgs=  1
P:  Vendor=0409 ProdID=0040 Rev= 1.21
S:  Manufacturer=NEC
S:  Product=NEC USB UF000x
C:* #Ifs= 1 Cfg#= 1 Atr=80 MxPwr=500mA
I:  If#= 0 Alt= 0 #EPs= 3 Cls=08(stor.) Sub=04 Prot=00 Driver=usb-storage
E:  Ad=82(I) Atr=02(Bulk) MxPS=  64 Ivl=0ms
E:  Ad=01(O) Atr=02(Bulk) MxPS=  64 Ivl=0ms
E:  Ad=83(I) Atr=03(Int.) MxPS=   2 Ivl=127ms

US_SC_UFI == Sub=04 
US_PR_CBI == Prot=00

diff -p -purN linux-2.6.5/drivers/usb/storage/unusual_devs.h linux-2.6.5-bug42892/drivers/usb/storage/unusual_devs.h
--- linux-2.6.5/drivers/usb/storage/unusual_devs.h	2004-09-17 11:23:42.000000000 +0200
+++ linux-2.6.5-bug42892/drivers/usb/storage/unusual_devs.h	2004-09-17 11:34:09.000000000 +0200
@@ -73,6 +73,15 @@ UNUSUAL_DEV(  0x03f0, 0x0307, 0x0001, 0x
 		US_SC_8070, US_PR_SCM_ATAPI, init_8200e, 0), 
 #endif
 
+/*
+ * FSC BX600 have this USB floppy (bug #42892 olh@suse.de)
+ */
+UNUSUAL_DEV(  0x0409, 0x0040, 0x0000, 0xffff,
+		"NEC",
+		"NEC USB UF000x",
+		US_SC_UFI, US_PR_CBI, NULL,
+		US_FL_SINGLE_LUN ),
+
 /* Deduced by Jonathan Woithe <jwoithe@physics.adelaide.edu.au>
  * Entry needed for flags: US_FL_FIX_INQUIRY because initial inquiry message
  * always fails and confuses drive.
